suite-native/navigation/src/components/Screen.tsx

```diff
@@ -87,13 +88,14 @@ export const Screen = ({
     customVerticalPadding = nativeSpacings.small,
     customHorizontalPadding = nativeSpacings.small,
     extraKeyboardAvoidingViewHeight = 0,
+    hasBottomInset = true,
 }: ScreenProps) => {
     const {
         applyStyle,
         utils: { colors, isDarkColor },
     } = useNativeStyles();
 
-    const isTabBarVisible = !!useContext(BottomTabBarHeightContext);
+    const isTabBarVisible = !!useContext(BottomTabBarHeightContext) || !hasBottomInset;
```

Comment:
Please do not append the `hasBottomInset` condition to the `isTabBarVisible` const, because it does not have anything to do with tab bar. Maybe change to something like this:

```tsx

const isTabBarVisible = !!useContext(BottomTabBarHeightContext)

...

style={applyStyle(screenContainerStyle, {
                ...,
                hasBottomInset: hasBottomInset || !isTabBarVisible,
            })}
```
Maybe changing the name of the style prop would make a sense. I just wanted ilustrate my point.

Reply:
The condition didn't make a lot of sense anyway. Refactored it a little bit and now it should be a little bit clearer.
https://github.com/trezor/trezor-suite/pull/9450/commits/c1af1be0c3c4fb06910b7dc95fdc10bcaa08b59a